sdks/python/apache_beam/runners/common.py

```diff
@@ -703,7 +703,8 @@ def _reraise_augmented(self, exn):
       raise
     step_annotation = " [while running '%s']" % self.step_name
     # To emulate exception chaining (not available in Python 2).
-    original_traceback = sys.exc_info()[2]
+    if isinstance(exn.args[0], BaseException):
+      exn = exn.args[0]
```

Comment:
In Python 2 the arguments to the exception was available by iterating over the exception. In Python 3, the arguments are contained in the `.args` attribute.